docs/02.API-REFERENCE.md

```diff
@@ -7007,28 +6982,25 @@ main (void)
 - [jerry_get_internal_property](#jerry_get_internal_property)
 
 
-## jerry_init_property_descriptor_fields
+## jerry_make_empty_property_descriptor
```

Comment:
This is a new function and should be documented with `New in .... release` (please see other "new" api methods for the format of this). Also I was wondering if we want to mention if the API replaces another method? That is adding an entry like `Replaces .... api method`.